librz/util/sys.c

```diff
@@ -603,6 +615,8 @@ RZ_API int rz_sys_cmd_str_full(const char *cmd, const char *input, char **output
 		.stdin_pipe = input ? RZ_SUBPROCESS_PIPE_CREATE : RZ_SUBPROCESS_PIPE_NONE,
 		.stdout_pipe = output ? RZ_SUBPROCESS_PIPE_CREATE : RZ_SUBPROCESS_PIPE_NONE,
 		.stderr_pipe = sterr ? RZ_SUBPROCESS_PIPE_CREATE : RZ_SUBPROCESS_PIPE_NONE,
+		.fork_mode = RZ_SUBPROCESS_FORK,
```

Comment:
I see there are many of these cases. Now, or in the future, it might make sense to create a `rz_subprocess_opt_init` to avoid this and prevent errors from uninitialized fields.

Reply:
If not now, please create an issue for this.

Reply:
To be fair, `RZ_SUBPROCESS_FORK` has the value 0. So even if we don't initialize the `fork_mode` and `pty` field explicitly, we would still get `fork_mode = RZ_SUBPROCESS_FORK` and `pty = NULL`. I just explicitly put them everywhere for the sake of it.

Reply:
Creating an `rz_subprocess_opt_init` does not make sense to me because all it would do is just take every argument and add it to a `RzSubprocessOpt` struct. I would rather have struct initialization than a 9 argument function (which would be very ugly)>

Reply:
I didn't mean to put all fields as args. What we have done for similar cases is to put common options as arguments, and then the caller will setup the extra/special fields. By doing so you can ensure all fields are initialized correctly.